roles/nv_gpu/tasks/install_nv.yml

```diff
@@ -48,18 +48,17 @@
 
 - name: "Find the GPU Operator OperatorHub InstallPlan"
   command:
-    oc get installplan
-      -oname
-      -loperators.coreos.com/gpu-operator-certified.openshift-operators
-      -n openshift-operators
+    oc get Subscription/gpu-operator-certified
```

Comment:
I like this patch, could you add a `#TODO` to remove this once we get into 4.6-4.7-.4.8
4.5 was using a diff strategy for Operatorhub, that is now default since 4.6, so we might be able to go back to this way.